packages/micropip/src/micropip/micropip.py

```diff
@@ -263,7 +276,7 @@ def find_wheel(self, metadata, req: Requirement):
                 if _is_pure_python_wheel(fileinfo["filename"]):
                     return fileinfo, ver
 
-        raise ValueError(f"Couldn't find a pure Python 3 wheel for '{req}'")
+        return None, None
```

Comment:
Could you please also add type annotations for the output, and a docstring explaining in particular when you would get the `None, None` output? We use the [numpy docstring style](https://numpydoc.readthedocs.io/en/latest/format.html)

Not sure why those methods were not documented, but it's probably a good idea to add some docs at least for this one.